sites/public-docsite-v9/package.json

```diff
@@ -7,22 +7,18 @@
     "build": "just-scripts build",
     "build-storybook": "build-storybook -s ./public -o ./dist/storybook --docs",
     "chromatic": "npx chromatic@6.4.3 --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes --build-script-name build-storybook",
-    "clean": "just-scripts clean",
-    "code-style": "just-scripts code-style",
-    "just": "just-scripts",
-    "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook"
+    "storybook": "start-storybook -p 6006 -s public",
+    "storybook:docs": "yarn storybook --docs"
   },
   "devDependencies": {
-    "@fluentui/eslint-plugin": "*",
```

Comment:
without these, things will not work as those versions: 
- are invalid (not published to npm)
- some packages are private